app/guid-node/files/provider/template.hbs

```diff
@@ -11,7 +11,65 @@
             {{!-- Placeholder for header --}}
         </layout.heading>
         <layout.leftNavOld as |leftNav|>
-            {{!-- Placeholder for leftnav links --}}
+            <leftNav.link
+                data-test-overview-link
+                data-analytics-name='Overview'
+                @href='/{{this.model.node.guid}}/'
+                @icon='home'
+                @label={{t 'node.left_nav.overview'}}
+            />
+            <leftNav.link
+                data-test-files-link
+                data-analytics-name='Files' 
+                @route='guid-node.files'
+                @models={{array this.model.node.guid}}
+                @icon='file-alt'
+                @label={{t 'node.left_nav.files'}}
+            />
```

Comment:
We should also have the list of available providers here with the current one highlighted, or we should make a new ticket to handle this functionality.

Reply:
I think a separate ticket for this and also some left-nav related styling would be preferable

Reply:
https://openscience.atlassian.net/browse/ENG-3761